### PR TITLE
Nextcloud Apps guides and refactor

### LINKED ISSUE #313: [maint] Improve Nextcloud Docs
- [ ] Add pictures, etc.  Simplify where possible.  Possibly needs an index.
- [x] Add Memories instructions
    1. Install Memories from NC Apps page
    2. When complete, hit the icon in the top menu to visit the app, you will see this image:
    image
    3. Go to Nextcloud service page -> Actions -> Index Media for Memories App (and execute it)
    4. Following success screen ::image:: , return to Memories, click "Start" and select your desired root media directory

### PR DESCRIPTION
**Pull request by kn0wmad**
Closes #313

### CODE REVIEW

File: site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-cookbook.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+..................................
+There are 2 versions of this app available on Android, and one on iOS.  Check this `repository <https://github.com/nextcloud/cookbook/>`_ for details.
+
+#. Go to your app store of choice and install the Nextcloud Cookbook app.
+
+#. (Android connection) You can select a previously connected Nextcloud account, or scan an app-specific QR, created under Nextcloud's main menu -> Personal Settings -> Security
+
+#. (iOS connection) You will be prompted to enter the following:
+
+    - Server URL - get this from your server -> Nextcloud -> Interfaces (LAN is recommended)
```
Comment (gStart9) [inline site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-cookbook.rst:39]: "... from your server's Services -> Nextcloud ..."

File: site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-cookbook.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    - Username - get this from your server -> Nextcloud -> Properties (embassy is default)
```
Comment (gStart9) [inline site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-cookbook.rst:40]: "... from your server's Services -> Nextcloud ..."

File: site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-cookbook.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    - Password - get this from your server -> Nextcloud -> Properties
```
Comment (gStart9) [inline site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-cookbook.rst:41]: "... from your server's Services -> Nextcloud ..."

File: site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-notes.rst [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+Desktop & Mobile Setups
+-----------------------
+The real beauty of Notes is being able to sync to your mobile devices for use on the go.
+
+Nextcloud Notes (Android)
+.........................
+#. Download the Nextcloud Notes app from your favorite app store (Aurora, F-Droid, or Play Store).
+
+#. On the opening screen, select your existing Nextcloud account.  If one does not exist, make sure to start with your :ref:`client setup<nextcloud-setup>` or you can hit "Add Account," then create an app passcode QR under Nextcloud's main top-right menu -> Personal Settings -> Security and scan it to grant access.
```
Comment (gStart9) [inline site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-notes.rst:37]: top-right menu -> Personal Settings -> Security > Devices & Sessions > Create new app password

File: site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-photos.rst [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+Desktop & Mobile Setups
+-----------------------
+
+Les Pas (Android)
+.................
+`Les Pas <https://github.com/scubajeff/lespas>`_ is a great mobile gallery for viewing and organizing your photos, gifs, and videos.
+
+#. First, install Les Pas from your favorite app store (Aurora, F-Droid, or Play Store), or from `the github repository <https://github.com/scubajeff/lespas/releases>`_.
+
+#. At the welcome screen, you will be asked to either enter your server URL or scan a QR code.  In your Nextcloud instance, click the top right-hand menu -> Personal Settings -> Security and create an app password.  Hit "show QR Code" and then scan it with Les Pas.
```
Comment (gStart9) [inline site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-photos.rst:26]: top-right menu -> Personal Settings -> Hamburger (3 lines) Menu > Security > Devices & Sessions > Create new app password

File: site/source/user-manual/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst [hunk 6]
```diff
@@ -55,7 +55,7 @@ You will first need to have the :ref:`Tor daemon running<tor-linux>`.
 
 1. On your desktop application. Click the account in the top left -> Settings, then in Settings, click Network, then "Specify proxy manually as" and "SOCKS5 proxy." Enter "127.0.0.1" for the Host and "9050" for the port.
 2. Close the Settings screen and click the account in the top left again, then "Add Account."
-3. On the following screen, click "Log in," then enter your Nextcloud Tor server address, which you can copy from the Nextcloud page on your Embassy -> Interfaces - Tor. This must start with `http://` and end with .onion. Click Next.
+3. On the following screen, click "Log in," then enter your Nextcloud Tor server address, which you can copy from the Nextcloud page on your Embassy -> Interfaces -> Tor. This must start with `http://` and end with `.onion`. Click Next.
```
Comment (gStart9) [inline site/source/user-manual/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst:58]: It's not this simple, but that's what my upcoming changes will be about so this is fine for now

**Review by gStart9 — COMMENTED** [review]
See comments, otherwise looks good.

File: site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-cookbook.rst (hunk 1 above)
Comment (gStart9) [inline site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-cookbook.rst:39]:
```suggestion
    - Server URL - get this from your server's Services -> Nextcloud -> Interfaces (LAN is recommended)
```

File: site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-cookbook.rst (hunk 2 above)
Comment (gStart9) [inline site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-cookbook.rst:40]:
```suggestion
    - Username - get this from your server's Services -> Nextcloud -> Properties (embassy is default)
```

File: site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-cookbook.rst (hunk 3 above)
Comment (gStart9) [inline site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-cookbook.rst:41]:
```suggestion
    - Password - get this from your server's Services -> Nextcloud -> Properties
```

File: site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-notes.rst (hunk 4 above)
Comment (gStart9) [inline site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-notes.rst:37]:
```suggestion
#. On the opening screen, select your existing Nextcloud account.  If one does not exist, make sure to start with your :ref:`client setup<nextcloud-setup>` or you can hit "Add Account," then create an app passcode QR under Nextcloud's main top-right menu -> Personal Settings -> Hamburger (3 lines) Menu > Security > Devices & Sessions > _Create new app password_ and scan it to grant access.
```

File: site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-cookbook.rst (hunk 2 above)
Comment (gStart9) [inline site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-cookbook.rst:40]:
```suggestion
    - Username - get this from your server's Services -> Nextcloud -> Properties (embassy is default)
```

File: site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-cookbook.rst (hunk 1 above)
Comment (gStart9) [inline site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-cookbook.rst:39]:
```suggestion
    - Server URL - get this from your server's Services -> Nextcloud -> Interfaces (LAN is recommended)
```

File: site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-photos.rst (hunk 5 above)
Comment (gStart9) [inline site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-photos.rst:26]:
```suggestion
#. At the welcome screen, you will be asked to either enter your server URL or scan a QR code.  In your Nextcloud instance, click the top-right menu -> Personal Settings -> Hamburger (3 lines) Menu > Security > Devices & Sessions > _Create new app password_.  Hit "show QR Code" and then scan it with Les Pas.
```

**Review by gStart9 — COMMENTED** [review]
Now it's ready for your reviewal

File: site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-notes.rst (hunk 4 above)
Comment (kn0wmad) [reply site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-notes.rst:37]: Is this on mobile?  I hadn't considered that there were 2 different operations

**Review by gStart9 — APPROVED** [review]

**Comment by gStart9** [comment]
It should be explicit in these instructions - because they are instructions for android.

**Pull request merged**